core/playback/mpv/track.go

```diff
@@ -104,6 +105,11 @@ func (t *MpvTrack) Pause() {
 }
 
 func (t *MpvTrack) Close() {
+	// Windows automatically handles closing
+	// and cleaning up named pipe
+	if runtime.GOOS == "windows" {
```

Comment:
It is not a good practice to use `runtime.GOOS` in your files (unless it is a really small check). Usually we create one file for each variation and use build tags to control which version to use.

For an example, see `scanner/metadata/taglib/get_filename.go` and `scanner/metadata/taglib/get_filename_win.go`

In your case I'd have two `Close()` methods in different files, one for windows (do nothing) and the other for all other platforms with the actual code.

Let me know if it makes sense to you or if you want me to do that.

Reply:
yeah was wondering about this and hoping to get feedback on what best practice is for this project. makes sense and thank you for the examples - will give it a shot and report back